### PR TITLE
Kata manager fix install

### PR DESCRIPTION
**Pull request by jodh-intel**
Fix installation of both Kata and containerd for the `kata-manager.sh` script and update the docs.

Fixes: #3674.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: utils/kata-manager.sh [hunk 1]
```diff
@@ -543,31 +608,72 @@ test_installation()
...
+			o) only_kata="true" ;;
+			r) cleanup="false" ;;
+		esac
+	done
+
+	shift $[$OPTIND-1]
 
-	local kata_version="${1:-}"
-	local containerd_version="${2:-}"
+	[ -z "$kata_version" ] && kata_version="${1:-}"
```
Comment (devimc) [inline utils/kata-manager.sh:670]: Q. can this fail with `set -e` ?  for this kind of checks I think the best  option is `-n` and `||` , isn't it? :thinking:

**Review by devimc — APPROVED** [review]
thanks @jodh-intel - just one question

File: utils/kata-manager.sh (hunk 1 above)
Comment (jodh-intel) [reply utils/kata-manager.sh]: I think it's correct, but given the shell's obscure rules I've added `|| true` "just in case" ;)

**Review by fidencio — APPROVED** [review]
lgtm, with the caveat that I am very much **not** familiar with kata manager.

**Comment by jodh-intel** [comment]
@devimc - ptal.

**Comment by jodh-intel** [comment]
Could we get 1 more ack on this folks? We really need this to land asap.

**Review by devimc — APPROVED** [review]
thanks @jodh-intel

**Pull request merged**